js/tests/unit/carousel.spec.js

```diff
@@ -479,6 +479,22 @@ describe('Carousel', () => {
         }, 10)
       })
     })
+
```

Comment:
I think you could add some tests : 
- For the behavior of the carousel on hover with a play/pause button
- With a button that is placed anywhere but after the carousel (in the carousel or not a sibling)